pkg/paravirt/paravirt_protected_entity_type_manager.go

```diff
@@ -195,13 +195,13 @@ func (this *ParaVirtProtectedEntityTypeManager) CreateFromMetadata(ctx context.C
 
 	// Get backupRepository from Guest, set backRepositoryName to backupRepositoryObj.SvcBackupRepositoryName
 	var backupRepo *snapshotUtils.BackupRepository
-	if backupRepositoryName != "" && backupRepositoryName != utils.WithoutBackupRepository {
+	if backupRepositoryName != "" && backupRepositoryName != "without-backup-repository" {
```

Comment:
We can make it a constant. It should be good if constants.go does not consume any other modules in this project.